src/commands/Dev/MaintenanceCommand.ts

```diff
@@ -1,40 +1,43 @@
 import CommandContext from '@structures/CommandContext';
 import MenheraClient from 'MenheraClient';
 import Command from '@structures/Command';
+import { Message } from 'discord.js';
 
 export default class MaintenanceCommand extends Command {
   constructor(client: MenheraClient) {
     super(client, {
       name: 'maintenance',
       aliases: ['cmd'],
-      description: 'Coloca ou tira um comando de manutenção',
       devsOnly: true,
       category: 'Dev',
     });
   }
 
-  async run(ctx: CommandContext) {
+  async run(ctx: CommandContext): Promise<Message> {
```

Comment:
every command must return a Promise<void> or void

Reply:
But it already return a promise